api/docs/http_catalog.md

```diff
@@ -60,10 +59,10 @@ Optional parameters that may be provided to the POST or GET:
 
     POST /puppet/v3/catalog/elmo.mydomain.com
 
-    environment=env&configured_environment=canary_env&facts_format=pson&facts=%7B%22name%22%3A%22elmo.mydomain.com%22%2C%22values%22%3A%7B%22architecture%22%3A%22x86_64%22%7D&transaction_uuid=aff261a2-1a34-4647-8c20-ff662ec11c4c
+    environment=env&configured_environment=canary_env&facts_format=application%2Fjson&facts=%7B%22name%22%3A%22elmo.mydomain.com%22%2C%22values%22%3A%7B%22architecture%22%3A%22x86_64%22%7D&transaction_uuid=aff261a2-1a34-4647-8c20-ff662ec11c4c
```

Comment:
The facts string, when updated to reflect double encoding, is:
```
"%257B%2522name%2522%253A%2522elmo.mydomain.com%2522%252C%2522values%2522%253A%257B%2522architecture%2522%253A%2522x86_64%2522%257D%257D"
```
I.e.,
```
JSON.parse(CGI.unescape(CGI.unescape(doubly_escaped)))
=> {"name"=>"elmo.mydomain.com", "values"=>{"architecture"=>"x86_64"}}
```
Perhaps worth updating for accuracy